.github/workflows/build_and_deploy.yaml

```
@@ -0,0 +1,41 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [master, feat/nextjs-landing]
```

We should leave only master before merge this branch
```suggestion
    branches: [master]
```